openmetadata-ui/src/main/resources/ui/src/utils/Alerts/AlertsUtil.tsx

```diff
@@ -374,6 +374,30 @@ export const getConnectionTimeoutField = () => (
   </>
 );
 
+export const getReadTimeoutField = () => (
+  <>
+    <Row align="middle">
+      <Col span={7}>{`${t('label.read-type', {
+        type: t('label.timeout'),
+      })} (${t('label.second-plural')})`}</Col>
+      <Col span={1}>:</Col>
+      <Col data-testid="read-timeout" span={16}>
+        <Form.Item name="readTimeout">
+          <Input
+            data-testid="read-timeout-input"
+            defaultValue={10}
+            placeholder={`${t('label.read-type', {
+              type: t('label.timeout'),
+            })} (${t('label.second-plural')})`}
+            type="number"
+          />
+        </Form.Item>
+      </Col>
+    </Row>
+    <Divider className="p-x-xs" />
```

Comment:
Let's not add one more Divider, better to keep both timeout fields in a single block.